go.mod

```diff
@@ -5,68 +5,67 @@ go 1.21
 toolchain go1.21.4
 
 require (
-	github.com/attestantio/go-eth2-client v0.19.8
-	github.com/decred/dcrd/dcrec/secp256k1/v4 v4.2.0
+	github.com/attestantio/go-eth2-client v0.19.7
+	github.com/decred/dcrd/dcrec/secp256k1/v4 v4.3.0
 	github.com/ferranbt/fastssz v0.1.3
 	github.com/gorilla/handlers v1.5.2
 	github.com/gorilla/mux v1.8.1
 	github.com/jonboulle/clockwork v0.4.0
-	github.com/libp2p/go-libp2p v0.31.0
-	github.com/obolnetwork/charon v0.19.0
-	github.com/rs/zerolog v1.31.0
+	github.com/libp2p/go-libp2p v0.33.2
+	github.com/obolnetwork/charon v0.19.2
+	github.com/rs/zerolog v1.32.0
 	github.com/spf13/cobra v1.8.0
 	github.com/spf13/pflag v1.0.5
 	github.com/spf13/viper v1.18.2
-	github.com/stretchr/testify v1.8.4
-	golang.org/x/sync v0.6.0
-	google.golang.org/protobuf v1.32.0
+	github.com/stretchr/testify v1.9.0
+	golang.org/x/sync v0.7.0
+	google.golang.org/protobuf v1.33.1-0.20240408130810-98873a205002
```

Comment:
Why this version and not stable one?

Reply:
not sure, i let `go get -u` do it's thing :/